src/main/java/io/jsonwebtoken/JwtParser.java

```diff
@@ -261,8 +261,37 @@
      * otherwise.
      */
     boolean isSigned(String jwt);
+    
+    /**
+     * Do not validate the {@code exp} claim when parsing the JWT.
+     * <p>
+     * <p>Note that this circumvents security features of JWT.</p>
+     *
+     * @return the parser for method chaining.
+     * @see ExpiredJwtException
+     */
+    JwtParser ignoreExpiry();
 
     /**
+     * Do not validate the {@code nbf} claim when parsing the JWT.
+     * <p>
+     * <p>Note that this circumvents security features of JWT.</p>
+     * @return the parser for method chaining.
+     * @see PrematureJwtException
+     */
+    JwtParser ignoreNotBefore();
+
+    /**
+     * Do not validate the JWS digital signature.
+     * <p>
+     * <p>Note that this circumvents security features of JWT and JWS.</p>
+     *
+     * @return the parser for method chaining.
+     * @see SignatureException
+     */
+    JwtParser ignoreSignature();
```

Comment:
I think calling this method by parsing the token is obvious enough. 
The way suggest (catching exception and ignoring it) is not really a good idea. Lets take a look in following scenario:

- We want to ignore expiry but do all other validations (nbf, signature, etc.)
- If we catch ExpiredJwtException we can surely say the token is expired, but other validations will not be executed, because with thrown exception, execution of parsing will be stopped immediately. So we can not ensure if nbf claim is valid or not. Currently signature validation comes first, but in the future the implementor may decide to change the order of validations. In this case we can also not sure if token signature valid at all. 

So we need a mechanism to say explicitly, to that validations but this not. If you see this use case not a common one. We can stop the discussion here :)